interpreter/text/arrange.ml

```diff
@@ -481,13 +481,13 @@ let rec instr e =
     | ReturnCallIndirect (x, y) ->
       "return_call_indirect " ^ var x, [Node ("type " ^ var y, [])]
     | ContNew x -> "cont.new", [Node ("type " ^ var x, [])]
-    | ContBind x -> "cont.bind", [Node ("type " ^ var x, [])]
+    | ContBind (x, y) -> "cont.bind", [Node ("type " ^ var x, []); Node ("type " ^ var y, [])]
```

Comment:
There 's some inconsistency within Wasm, but I don't think we should require the `(type $t)` here, but plain `$t` should do. Same for `cont.new`.

Reply:
OK!